yarn.lock

```diff
@@ -3049,14 +3049,6 @@
     "@typescript-eslint/typescript-estree" "4.14.1"
     debug "^4.1.1"
 
```

Comment:
I dont why these appeared, I have latest from master, I deleted the `node_modules` folder and hit `yarn`